app/views/layouts/application.html.erb

```diff
@@ -10,14 +10,16 @@
   </head>
 
   <body>
-  <% flash.each do |type, message| %>
-    <section class=<%= type %>>
-      <p class="user-message"><%= message %></p>
-    </section>
-  <% end %>
-  <nav class="navbar navbar-expand-lg navbar-light fixed-top" id="mainNav">
-    <div class="container">
-        <a class="navbar-brand js-scroll-trigger" href="/">I Scream Arcade</a>
+    <nav class="navbar navbar-expand-lg navbar-light fixed-top" id="mainNav">
+      <a class="navbar-brand js-scroll-trigger" href="/">
+        <img class="icon" src="https://cdn2.iconfinder.com/data/icons/funeral-filloutline/64/scythe-reaper-cultures-spooky-terror-scary-fear-horror-death-halloween-256.png" alt="zombie" border="0"></a>
```

Comment:
This image name `spooky-terror-scary-fear-horror-death` is just great. And fits the theme. I'm down for this logo

Reply:
haha, yeah I like that. I enjoy the 64 bit nature